src/platforms/python/common/configuration/integrations/index.mdx

```diff
@@ -1,10 +1,34 @@
 ---
 title: Integrations
 description: "Sentry provides additional integrations designed to change configuration or add instrumentation to your application."
-sidebar_order: 700
+sidebar_order: 30
 ---
 
-Sentry provides a number of additional integrations which are designed to change configuration or add additional instrumentation to your application.
+Sentry provides a number of integrations which are designed to add automatic instrumentation to your application.
+
+Integrations can be configured in the call to `sentry_sdk.init()` like this:
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+
+sentry_sdk.init(
+    ...
+    integrations=[
+        DjangoIntegration(),
+        CeleryIntegration(),
+        RedisIntegration(),
+    ],
+)
+```
+
```

Comment:
This appears to duplicate similar content that already exists on the child pages of this page and I don't think it needs to be repeated. Take a look at the JavaScript Integrations page for an example of what this page generally looks like. I think the prose is that you've added is fine, but the code snippet seems out of place.

Reply:
It is a similar content as in the child pages, but it is illustrating, that one can add multiple integrations (which is not shown elsewhere) and also it enables Django, Celery and Redis which are possibly the most used combinations for python web projects and there it can be kind of a shortcut for users doing a lot of Django projects. They just need to copy/paste this and add the dsn and they are done

Reply:
@untitaker, your opinion on this would be helpful here. I'm concerned about this Integrations page being very different from other ones. Do you think this information is important enough to keep on this page vs (perhaps) having it as an extra code snippet on the child pages?

Reply:
The main thing why I put this here is that developers are lazy and don't want to think about integrating third party libraries. If your are doing a new Django project you can just go to "Configuration" -> "Integrations" then copy this snipped and you are done. 
But because we have also automatic enabling of integrations this is not necessary at all. (we do not have documented automatic enabling of integrations anywhere, but this is another discussion for another place)

So I am also fine with removing it.

Reply:
I have now removed the code snippet. After reading the docs guide and thinking about it, it is really a duplicated code snippet.